src/qibolab/_core/instruments/qblox/results.py

```diff
@@ -39,6 +39,7 @@ def integration_lenghts(
             (mod_id, i): seq.integration_length_acq()
             for mod_id, mod in modules.items()
             for i, seq in enumerate(mod.sequencers)
+            if hasattr(seq, "integration_length_acq")
```

Comment:
This seems to be needed because non-QRM sequencers do not have this attribute and it gives an `AttributeError`. A different potential fix is to loop over QRMs only here.